spec/features/requesting_new_allocations_spec.rb

```diff
@@ -270,4 +291,45 @@ def when_i_search_for_a_training_provider_with_empty_string
   def and_i_see_error_message_that_i_must_add_more_info
     expect(page).to have_content("You need to add some information")
   end
+
+  def when_i_fill_in_the_number_of_places_input
+    find("#number-of-places-input").fill_in(with: "2")
```

Comment:
We should use a page object for these so as we're consistent with the other specs. Also happy to discuss whether we still want to use SitePrism but it does stop the HTML structure of the pages leaking across specs which I think is generally a good thing.